### DISCUSSION TITLE
[Feature] Ability to search for text in images

### DISCUSSION THREAD

**Discussion by 569258yin** [immich-app/immich#13904] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I want a function that can fuzzily search for text in pictures, similar to the search box function in the iPhone album, but can search for text that appears in the picture. I have conceived the execution logic of the program:
1. After uploading, use the OCR tool to recognize the content of the current text
2. Build the text content into the search engine and associate it with the original picture ID
3. When searching, go to the search engine for fuzzy search and give the picture with the associated ID

Do the developer have plans to introduce this function? If not, are developers allowed to integrate it after development?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3168
